lib/roo_on_rails/checks/documentation/playbook.rb

```diff
@@ -0,0 +1,23 @@
+require 'roo_on_rails/checks/base'
+
+module RooOnRails
+  module Checks
+    module Documentation
+      class Playbook < Base
+        LOCATION = 'PLAYBOOK.md'.freeze
+
+        def intro
+          'Looking for an on-call Playbook...'
+        end
+
+        def call
+          if File.exist?(LOCATION)
```

Comment:
I was thinking that, I figured the first PR to make this a requirement would be good, then a second that we can argue over for the structure :P